### ISSUE TITLE
Platypus: non-unique `nav` landmark

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#473]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the page has `nav` landmarks which are lacking an accessible name (unique role or role/label/title combination).

---

https://dequeuniversity.com/rules/axe/4.3/landmark-unique?application=axeAPI

---

**User impact:** Moderate
**Disabilities affected:** Sighted Keyboard Users, Blind, Deafblind
**Requirement(s):** Best Practice
**WCAG Success Criteria:** N/A

**Comment by micpap25** [comment]
Hi, the nav landmarks referred to here are part of the sidebar which is present on all pages in Platypus... This seems like a pretty straightforward fix, just noting it is not specific to this page. I will look into this and come up with `aria` labels for the sidebar's content.

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
